common/php/class-module.php

```diff
@@ -7,6 +7,7 @@
 
 if ( !class_exists( 'EF_Module' ) ) {
 
+#[\AllowDynamicProperties]
```

Comment:
Is this attribute necessary since I see that the properties are later being defined?